src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -269,7 +310,16 @@ class DetailsSidebar extends React.PureComponent<Props, State> {
             bannerPolicy,
         }: Props = this.props;
 
-        const { accessStats, accessStatsError, file, fileError, isLoading }: State = this.state;
+        const {
+            accessStats,
+            accessStatsError,
+            classificationInfo,
```

Comment:
nitpick, can we just call it classification to match accessStats? or change accessStats to accessStatsInfo

Reply:
Deal!